app/views/users/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".title" %></h1>
+
+<%= will_paginate %>
+
+<ul class="users">
+  <%= render @users %>
```

Comment:
`render` ở đây hoạt động như nào?

Reply:
render ở đây sẽ tìm 1 partial  _user.html.erb sau đó for qua từng user và render _user.html.erb

Reply:
ok